openwisp_firmware_upgrader/filters.py

```diff
@@ -0,0 +1,31 @@
+from django.utils.translation import gettext_lazy as _
+
+from openwisp_users.multitenancy import MultitenantOrgFilter
+
+from .swapper import load_model
+
+Build = load_model('Build')
+Category = load_model('Category')
+
+
+class CategoryFilter(MultitenantOrgFilter):
```

Comment:
This doesn't work as expected, I believe this should inherit MultitenantRelatedOrgFilter.

If you try this filter with a staff user which is org manager but not super admin you should see the list of categories empty even if you create categories for the org the user manages, for example, in my dev env this user has access to 2 orgs and there are existing categories in the DB: 

![Screenshot from 2022-12-13 15-53-46](https://user-images.githubusercontent.com/841044/207420432-05037a50-582d-4689-a9e1-e8c3debf0928.png)

But the filter is not showing any category:
![Screenshot from 2022-12-13 15-53-30](https://user-images.githubusercontent.com/841044/207420751-b016d220-7446-499c-a4ec-f80250ea9125.png)